### DISCUSSION TITLE
I can't add  a album or picture anymore

### DISCUSSION THREAD

**Discussion by TinusJan** [LycheeOrg/Lychee#2692] [Q&A]
Hi there, I'm a newby with Lychee, and have created a photosite with Lychee. I've created already 8 albums with photo's, but when trying to add a new album or uplod a phote got there was an error reporting. So nothing happens in my database. No photo add, no new album created.
Do I have to change some settings somewhere, what do I miss?

**Comment by ildyria** [comment]
It would be helpful if you provided us with the content of your Diagnostics. Because as is we really can't help you.

**Comment by TinusJan** [comment]
I'm a bit ashamed, but the problems seem to be solve. I've also changed settings in my PHP and, after one day waiting, this did the job. Now I can upload photo's again and add a new album again.
So problem is solved.
